nemo/collections/nlp/nm/trainables/dialogue_state_tracking/__init__.py

```diff
@@ -14,4 +14,4 @@
 # limitations under the License.
 # =============================================================================
 
-from nemo.collections.nlp.nm.trainables.dialogue_state_tracking.state_tracking_trade_nm import *
+from nemo.collections.nlp.nm.trainables.dialogue_state_tracking.trade_generator_nm import *
```

Comment:
please keep refactoring changes that do not fix bugs to a minimum at this point, only when absolute necessary.

Reply:
Such changes would not affect the QA as it is happening deep down and I think it is better to be done now before the release. Because we are going to train models after the QA and changes like this would break checkpoints.